tests/test_aes.py

```diff
@@ -0,0 +1,43 @@
+#
+# This file is part of pysnmpcrypto software.
+#
+# Copyright (c) 2018, Ilya Etingof <etingof@gmail.com>
+# License: http://snmplabs.com/pysnmp/license.html
+#
+import sys
+
+try:
+    import unittest2 as unittest
+
+except ImportError:
+    import unittest
+
+from pysnmpcrypto import aes
+
+
+class AesCaseBase(unittest.TestCase):
+    def setUp(self):
+        if sys.version_info[0] < 3:
```

Comment:
FYI for future reference, both Python 2 and Python 3 support the `b'some bytes'` syntax.